src/app/(landing-page)/blog/page.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import { useBlogs } from "@/api/blog";
+import BlogCard from "@/components/BlogPage/Cards/BlogCard";
+import { LoadingScreen } from "@/components/global-components";
+import { Container } from "@mui/material";
+
+const borderColor = "lightgrey";
+
+const BlogPage = () => {
+  const { blogs: posts } = useBlogs();
+
+  if (posts != undefined) {
```

Comment:
here i want you to return a loading state like the <LoadingScreen /> component or <CircularProgress /> if posts === undefined, and then render the rest of the content underneath outside the if statement

``
if (posts === undefined) return <LoadingScreen />;

return (
... your code here
)
``

Reply:
think i resolved this

Reply:
have you pushed changes? it doesn't look like its changed on the PR